.github/ci-files/federatedStorageConfigSecret.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 data:
-  federated-store.yaml: dHlwZTogUzMKY29uZmlnOgogIGJ1Y2tldDogImt1YmVjb3N0LVVOSVFVRV9OQU1FLW1ldHJpY3MiCiAgZW5kcG9pbnQ6ICJzMy5hbWF6b25hd3MuY29tIgogIHJlZ2lvbjogInVzLWVhc3QtMiIKICAjIHVzZSBpcnNhIHdoZW4gcG9zc2libGUtIG90aGVyd2lzZSBhY2Nlc3Mga2V5cyBjYW4gYmUgdXNlZDoKICAjIGFjY2Vzc19rZXk6ICJZT1VSX0tFWSIKICAjIHNlY3JldF9rZXk6ICJZT1VSX1NFQ1JFVCIKICBpbnNlY3VyZTogZmFsc2UKICBzaWduYXR1cmVfdmVyc2lvbjI6IGZhbHNlCiAgcHV0X3VzZXJfbWV0YWRhdGE6CiAgICAgICJYLUFtei1BY2wiOiAiYnVja2V0LW93bmVyLWZ1bGwtY29udHJvbCIKICBodHRwX2NvbmZpZzoKICAgIGlkbGVfY29ubl90aW1lb3V0OiA5MHMKICAgIHJlc3BvbnNlX2hlYWRlcl90aW1lb3V0OiAybQogICAgaW5zZWN1cmVfc2tpcF92ZXJpZnk6IGZhbHNlCiAgdHJhY2U6CiAgICBlbmFibGU6IHRydWUKICBwYXJ0X3NpemU6IDEzNDIxNzcyOA==
+  federated-store.yaml: FEDERATED_STORAGE_CONFIG_YAML
```

Comment:
Worth leaving a comment that the string `FEDERATED_STORAGE_CONFIG_YAML` gets replaced with an actual valid secret once the Github Action is run?